hips/draw/tests/test_simple.py

```diff
@@ -1 +1,52 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+import pytest
+from astropy.utils.data import get_pkg_data_filename
+from numpy.testing import assert_allclose
+
+from ..simple import make_sky_image, draw_sky_image
+from ...tiles import HipsSurveyProperties, HipsTileMeta, HipsTile
+from ...utils.testing import get_hips_extra_file, make_test_wcs_geometry, requires_hips_extra
+
+
+def get_test_tiles():
+    frames = dict({'equatorial': 'icrs', 'galactic': 'galactic', 'ecliptic': 'ecliptic'})
```

Comment:
This mapping between HiPS to SkyCoord strings for frame shouldn't be here in the test.
It should be available somewhere else, in a central place, and then if you need to call it from here (or just hard-code the values you need here in `get_test_tiles`.
Maybe putting the dict on the `HipsTileMeta` class would be a good location? Or somewhere in `hips.utils`?